lib/plug/parsers.ex

```diff
@@ -73,6 +73,9 @@ defmodule Plug.Parsers do
   the map of params parsed by one of the parsers listed in `:parsers` and
   `:params` set to the result of merging the `:body_params` and `:query_params`.
 
+  It is important that `Plug.Parsers` is ordered prior to the `:dispatch` plug
+  otherwise the matched function will not receive the parsed `Plug.Conn`.
```

Comment:
The issue is that `Plug.Parsers` can be used in any other plug, so it doesn't make much sense to mention on its documentation something about one of its many usages. Maybe we should rather change Plug.Router to provide more examples in there, maybe provide one with a full pipeline of common plugs, such as parsers, static, session and etc and explain where to put them (static before match, parsers and session before dispatch).